util/delete-cluster.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Delete the oldest cluster for a dataset.
+# Run after a redeploy.
+
+set -o errexit
+set -o nounset
+
+if (( $# != 1 ))
+then
+  echo "Usage: util/delete-cluster.sh <dataset>"
+  echo "  where <dataset> is the name of a directory in dataset_config/"
+  echo "Run this script from project root"
+  exit 1
+fi
+
+dataset=$1
+project_id=$(jq --raw-output '.project_id' dataset_config/${dataset}/deploy.json)
+
+gcloud config set project ${project_id}
+echo "gcloud project set to $(gcloud config get-value project)"
+
+# Need to get cluster name by sorting the list of clusters, and choosing to
+# use the one with the greatest timestamp (most recent)
+cluster_list=$(gcloud container clusters list)
+cluster_line=$(echo "${cluster_list}" | grep elasticsearch-cluster- | sort -rn -k1 | head -n1)
+cluster_name=$(echo "${cluster_line}" | awk '{print $1}')
```

Comment:
Delete cluster_name, it's unused.

To avoid these kinds of problems, don't use intermediate variables. Replace 26-28 with:
```
zone=$(gcloud container clusters list | grep elasticsearch-cluster | awk '{print $2}')
```
This also makes the code easier to read -- it's harder to read with intermediate variables.

Though in this case, you can use cluster_list intermediate variable if you want, since it's used later.